src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,41 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectTodo: () => Todo | undefined;
+  viewModule: (id :number) => void
+};
+
+export const TodoModal: React.FC<Props> = ({ selectTodo, viewModule }) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+
+  const selectedTodo = selectTodo();
+
+  if (selectedTodo === undefined) {
+    return null;
+  }
+
+  const {
+    userId,
+    id,
+    title,
+    completed,
+  } = selectedTodo;
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
What error do you have here? Why you need to use `eslint-disable`?
Could you please add the screen.